nimbus_nlp/NIMBUS_NLP.py

```diff
@@ -19,41 +6,45 @@
 # Temporary import for the classifier
 from nimbus_nlp.question_classifier import QuestionClassifier
 
-class NIMBUS_NLP:
 
-    @staticmethod
-    def predict_question(input_question):
-        '''
+# Made this an instantiable class to prevent the overhead of instantiating
+# a variable extractor and question classifier for every question.
+# Consider: Does this even need to be a class? Its functionality could be
+# moved to the Nimbus class of nimbus.py
```

Comment:
Good question! I agree. 

**{{REPLY}}**

and if @richagadgil chooses to make [csai-ml](https://github.com/calpoly-csai/csai-ml) into a **_pip-installable package_** then the class within `nimbus.py` can sufficiently act as a wrapper/interface for the _csai-ml_ code.